state/postgresql/postgresdbaccess.go

```diff
@@ -299,20 +325,19 @@ func (p *PostgresDBAccess) Get(parentCtx context.Context, req *state.GetRequest)
 	var (
 		value    []byte
 		isBinary bool
-		etag     uint64 // Postgres uses uint32, but FormatUint requires uint64, so using uint64 directly to avoid re-allocations
```

Comment:
as seen in lines below, are reallocations ok?

Reply:
It wasn't a choice as pgx doesn't support decoding a XID into uint64. Not sure if this allocation stays on the stack or escapes to the heap, however.